### PR TITLE
Kube 0.51

### PR DESCRIPTION
**Pull request by kflansburg**

### CODE REVIEW

**Comment by kflansburg** [comment]
Trying to keep up-to-date with `kube`, the functionality that is hopefully landing in the next release will be pretty useful for the Krator watcher API. 
https://github.com/clux/kube-rs/blob/master/CHANGELOG.md#unreleased

File: crates/krator/Cargo.toml [hunk 1]
```diff
@@ -50,8 +50,8 @@ tracing = { version = "0.1", features = ['log'] }
 tracing-futures = "0.2"
 
 [dev-dependencies]
-kube-derive = "0.48"
-schemars = "0.8.0"
+kube-derive = "0.51"
+schemars = "0.8.2"
```
Comment (thomastaylor312) [inline crates/krator/Cargo.toml:54]: Nit: Should this just be `0.8`?

**Review by thomastaylor312 — APPROVED** [review]
You keep beating me to updating this!

File: crates/krator/Cargo.toml (hunk 1 above)
Comment (kflansburg) [reply crates/krator/Cargo.toml:54]: Ah, yeah I guess so, I just wanted to be sure it updated because it patched a clippy lint.

**Pull request merged**